common/utils/include/claragenomics/utils/filesystem.hpp

```diff
@@ -0,0 +1,45 @@
+/*
+* Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+*
+* NVIDIA CORPORATION and its licensors retain all intellectual property
+* and proprietary rights in and to this software, related documentation
+* and any modifications thereto.  Any use, reproduction, disclosure or
+* distribution of this software and related documentation without an express
+* license agreement from NVIDIA CORPORATION is strictly prohibited.
+*/
+
+#pragma once
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <string>
+
+namespace claragenomics
+{
+
+namespace filesystem
+{
+
+bool dirExists(const std::string path)
```

Comment:
Why not just use [filesystem_exists](https://en.cppreference.com/w/cpp/filesystem/exists) ?